File: src/main/java/com/ject/studytrip/stamp/application/dto/StampsInfo.java
package com.ject.studytrip.stamp.application.dto;

import java.util.List;

public record StampsInfo(List<StampInfo> stampsInfos) {
    public static StampsInfo of(List<StampInfo> stampsInfos) {
        return new StampsInfo(stampsInfos);
    }
}

File: src/main/kotlin/com/ject/studytrip/stamp/application/dto/StampsInfo.kt
package com.ject.studytrip.stamp.application.dto

data class StampsInfo(
    val stampInfos: List<StampInfo>,
) {
    companion object {
        @JvmStatic
        fun of(stampInfos: List<StampInfo>): StampsInfo = StampsInfo(stampInfos)
    }
}
